incipyt/tools/venv.py

```diff
@@ -1,22 +1,23 @@
 import os
 
 from incipyt import commands, signals, tools
+from incipyt._internal.templates import FormatterEnviron
 
 
 class Venv(tools.Tool):
     """Scripts to add virtualenv to :class:`incipyt.project._Structure`."""
 
     def add_to_structure(self):
         """Add venv configuration to `project.structure`, do nothing."""
-        signals.vcs_ignore.emit(pattern=".env/")
+        signals.vcs_ignore.emit(pattern="{VENV_FOLDER}/")
 
     def pre(self, workon):
         """Run `python -m venv .env`.
 
         :param workon: Work-on folder.
         :type workon: :class:`pathlib.Path`
         """
-        env_path = workon / ".env"
+        env_path = workon / FormatterEnviron().format("{VENV_FOLDER}")
```

Comment:
I think this is a complicated way to write
```suggestion
        env_path = workon / project.environ["{VENV_FOLDER}"]
```

Don't forget to import project